WebDevStudios/wds-acf-blocks
# Linting Cleanup

Closes #3 

### DESCRIPTION ###
This PR fixes all the current linting issues of the repository. It is passing all current WDS Coding Standards via PHPCS, ESLint, Stylelint.

### SCREENSHOTS ###
![screenshot](https://i.imgur.com/hi3tAuP.jpg)

### OTHER ###

### STEPS TO VERIFY ###
Run the following commands in the terminal to verify the issues.

- `npm run lint:css`
- `npm run lint:js`
- `composer run lint`
